test/suites/dev/moonbase/test-xcm-v3/test-mock-hrmp-transact-ethereum-12.ts

```diff
@@ -82,6 +82,12 @@ describeSuite({
         let expectedTransferredAmount = 0n;
         let expectedTransferredAmountPlusFees = 0n;
 
+        // Just to make sure lazy state trie migration is done
+        // probably not needed after migration is done
+        for (let i = 0; i < 10; i++) {
+          await context.createBlock();
```

Comment:
@TarekkMA tis addition is invalid, the test fail because you forgot to change on_idle orders, you need to put the lazy migration pallet after the message queue pallet in construct_runtime macro